swap_meet/vendor.py

```diff
@@ -0,0 +1,77 @@
+
+class Vendor:
+
+    def __init__(self, inventory= None):
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+        
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+    
+
+    def get_by_category(self, category):
+        cat_list = []
+        for item in self.inventory:
+            if item.category is category:
+                cat_list.append(item)
+        return cat_list
+        #return [item for item in self.inventory if item.category == category]
+
+
+    def swap_items(self, other, my_item, their_item):
+        if my_item in self.inventory and their_item in other.inventory:
+            self.remove(my_item)
+            other.add(my_item)
+            self.add(their_item)
+            other.remove(their_item)
+            return True
+        else:
+            return False
+
+
+    def swap_first_item(self, other):
+        if len(self.inventory) == 0 or len(other.inventory) == 0:
```

Comment:
You can simplify this a bit.

```suggestion
        if not self.inventory or not other.inventory:
```